test/tests.js

```diff
@@ -274,6 +298,13 @@ QUnit.test('with attributes', function (assert) {
 	assert.strictEqual(document.cookie, '', 'should delete the cookie');
 });
 
+QUnit.test('deletion with unofficial attributes', function (assert) {
+	assert.expect(1);
+	var attributes = { path: '/', unofficial: 'anything' };
```

Comment:
> However you need to pass all official attributes I think, so I prefer to do it the same way.

About that, I don't think we need to pass all official attributes. According [to the docs](https://github.com/js-cookie/js-cookie/tree/7fd36e66b60a4e0ac9d09207fb3a348a2969152c#basic-usage), it's just the "exact same path and domain attributes that was used to set the cookie".

It would be nice to confirm if we need the "secure" flag too and document that!